### ISSUE TITLE
v0.10 e2e Tests Not Running Version v0.10 of Pipelines

### ISSUE THREAD

**Issue by danielhelfand** [tektoncd/cli#1193]
As discovered in #1189, it seems that e2e tests that test features not supported in v0.10.2 (e.g., finally logs, volumeclaimtemplates) are [running and succeeding in v0.10 e2e tests](https://github.com/tektoncd/cli/pull/1189#discussion_r496360628). There is nothing in logs to show what versions of pipeline is being used, so part of closing this issue should be to add something to the e2e process to display what version of pipelines/triggers is being used in tests. There should also be some report of whether tests are still appropriately running against v0.10.2.

**Comment by vdemeester** [comment]
@danielhelfand I think I know what happened, it's *relatively* recent (a month or so…). The commit culprit is https://github.com/tektoncd/plumbing/commit/b1c211c7ec3ff33c39ff02792400a33052f1e62f#diff-406185368ba7839d1459d3d51424f104 (and me :stuck_out_tongue_closed_eyes:). I forgot to add the environment variable back..

```
  - name: pull-tekton-cli-integration-tests-0_10
    labels:
      preset-presubmit-sh: "true"
    agent: kubernetes
    always_run: true
    decorate: true
    rerun_command: "/test pull-tekton-cli-integration-tests-0_10"
    trigger: "(?m)^/test (all|pull-tekton-cli-integration-tests-0_10),?(\\s+|$)"
    spec:
      containers:
      - image: gcr.io/tekton-releases/dogfooding/test-runner:latest
        imagePullPolicy: Always
        command:
        - /usr/local/bin/entrypoint.sh
        args:
        - "--scenario=kubernetes_execute_bazel"
        - "--clean"
        - "--job=$(JOB_NAME)"
        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
        - "--root=/go/src"
        - "--service-account=/etc/test-account/service-account.json"
        - "--upload=gs://tekton-prow/pr-logs"
        - "--" # end bootstrap args, scenario args below
        - "--" # end kubernetes_execute_bazel flags (consider following flags as text)
        - "./test/presubmit-tests.sh"
        - "--integration-tests"
```

**Comment by danielhelfand** [comment]
I am still seeing in recent prs that all tests are running against v0.10. If it is the case that these tests are running against a v0.10 pipelines, then certain tests should be skipped or fail.

**Comment by barthy1** [comment]
At this moment in plumbing Prow setup the `RELEASE_YAML` env variable points to `v0.10.2` pipeline - https://github.com/tektoncd/plumbing/blob/main/prow/config.yaml#L423, however test script from cli side expects `RELEASE_YAML_PIPELINES` and `RELEASE_YAML_TRIGGERS` - https://github.com/tektoncd/cli/blob/master/test/e2e-common.sh#L21 Can the mismatch between env variables be the reason of the observed behaviour?

**Comment by vdemeester** [comment]
I think this can be closed now, we don't run 0.10 e2e tests anymore.
/close
